gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Fri Oct 26 20:26:54 EDT 2018
```

Comment:
Any newly generated `gradle-wrapper.properties` files don't have this line anymore. I think it can be safely deleted.